distribution/sender/routes/shutdown-route.xml

```diff
@@ -6,8 +6,8 @@
             <method beanType="org.openmrs.eip.app.CustomFileOffsetBackingStore" method="disable()" />
         </script>
         
-        <!-- TODO Send a notification to the configured emails -->
-
+        <toD uri="direct:send-notification-email-route" />
```

Comment:
I think this needs to happen in a try finally block so that in case an error is encountered while sending the email, we are still able to shutdown the application

Reply:
Okay